docs/developer-docs/ai/overview.mdx

```diff
@@ -4,7 +4,7 @@ keywords: [intermediate, concept, AI, ai, deAI, deai]
 
 import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 
-# Decentralized AI overview
+# Decentralized AI Overview
```

Comment:
```suggestion
# Decentralized AI overview
```

Reply:
Capitalization is defined in the dev docs style guide: https://github.com/dfinity/portal/blob/master/style-guide.md#capitalization